driver/collector/postgres_collector.py

```diff
@@ -1005,3 +1020,12 @@ def _anonymize_query(row: Dict):
                 query = query.strip()
                 row["query"] = query.lower()
         return row
+
+    @staticmethod
+    def _add_logical_db_columns(results: Dict[str, any]) -> Dict[str, any]:
+        modded_results = {}
+        for logical_db_name in results:
+            # print(results[logical_db_name])
+            for data_item in results[logical_db_name]["data"]:
+                print(data_item)
```

Comment:
should we use 'logging' instead of 'print'?

Reply:
Yes, that's wip

Reply:
Fixed